### DISCUSSION TITLE
Can't set up dashboard in production

### DISCUSSION THREAD

**Discussion by eddyLazar** [saleor/saleor#6895] [Deployment]
### What I'm trying to achieve
I am trying to run dashboard and api in production mode. I set up droplet in digital ocean with Saleor instance https://demo.eddylazar.com/graphql/ here it has Saleor demo data. Then after successful login in dashboard I see this error
```
[{"errors": [{"message": "Unknown type \"Channel\".", "locations": [{"line": 1, "column": 29}], "extensions": {"exception": {"code": "GraphQLError"}}}, {"message": "Cannot query field \"channels\" on type \"Query\".", "locations": [{"line": 11, "column": 3}], "extensions": {"exception": {"code": "GraphQLError"}}}]}]
``` 

### Steps to reproduce the problem
1. go to https://eddy-demo-shop.netlify.app/
2. try to login with **login:** admin@example.com **pass:** admin.

### What I expected to happen
I expect that everything should work

### Screenshots

**System information**

Operating system:
Saleor version: using docker image mirumee/saleor:latest

**Comment by krzysztofwolski** [comment]
Seems like mismatched versions of Core and Dashboard.
